### DISCUSSION TITLE
Deno 1.5.2 now available 🎉

### DISCUSSION THREAD

**Discussion by brianleroux** [beginner-corp/begin-community#64] [Show and tell]
Keeping pace with the Deno team is a fun challenge but Begin.com now supports Deno 1.5.2 please let me know if you run into any issues!

**Comment by chisNaN** [comment]
Thanks Bryan
I have one but it is more related to typescript and not begin platform.
For instance you cannot do things like this or it will rise syntax error

`const o = await (await fetch('someJson')).json()`

you need to decouple

**Reply by brianleroux** [reply]
sorry I do not understand the question ?
